_posts/2017-01-13-xcode-8-fastlane-codesigning.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Xcode 8 Manual Codesigning with Fastlane"
+date: 2017-01-13 14:00
+author: ash
+categories: [mobile, ios, devops, ci]
+series: Open Source by Default
+---
+
+New year, new deploy process! Late last year our mobile team completed the update to Swift 3 (and thus, the update to Xcode 8). The latest version of Apple's IDE includes a lovely feature: automating provisioning profile management! (Note: not sarcasm, the feature is really nice.)
```

Comment:
Maybe put a link in about the automated provisioning profile ( or a screenshot?)